docs/changes.rst

```diff
@@ -106,6 +106,12 @@ Enhancements
 * Clarify documentation on wbem operation recorder in client.rst. see
   issue #741
 
+* Added an optional class path to the `CIMClass` class. The class path is
+  set in CIMClass objects returned by the GetClass, EmumerateClasses, and the
+  class-level Associators and References operations.  issue #349.
+
+* Added a ``host`` property to ``WBEMConnection`` which contains the host:port
+  component of the WBEM server's URL.  This helps addressing issue #349.
```

Comment:
I would add the comment about this not reflecting any change to the interaction with the host here.

Reply:
Done